overlord/snapstate/storehelpers.go

```diff
@@ -275,93 +272,6 @@ func downloadInfo(ctx context.Context, st *state.State, name string, revOpts *Re
 	return singleActionResult(name, action.Action, res, err)
 }
 
-func installInfo(ctx context.Context, st *state.State, name string, revOpts *RevisionOptions, userID int, flags Flags, deviceCtx DeviceContext) (store.SnapActionResult, error) {
```

Comment:
quite a few bits of this were not covered on master so I fear we might need to add a few tests to be sure that we didn't miss anything or discover that the code wasn't relevant